dev-requirements.txt

```diff
@@ -47,19 +41,44 @@ chardet==4.0.0
     #   requests
 click==7.1.2
     # via flask
+configparser==4.0.2
+    # via importlib-metadata
+contextlib2==0.6.0.post1
+    # via
+    #   importlib-metadata
+    #   importlib-resources
+    #   zipp
 cryptography==2.9.2
     # via cdispyutils
 distlib==0.3.1
     # via virtualenv
+git+https://github.com/uc-cdis/doiclient.git@1.0.0#egg=doiclient
+    # via
+    #   -r dev-requirements.in
+    #   indexd
+git+https://github.com/uc-cdis/dosclient.git@1.0.0#egg=dosclient
+    # via
+    #   -r dev-requirements.in
+    #   indexd
+enum34==1.1.10
+    # via cryptography
 filelock==3.0.12
     # via virtualenv
 flask==1.1.2
     # via
     #   cdiserrors
     #   cdispyutils
     #   indexd
+funcsigs==1.0.2
+    # via
+    #   mock
+    #   pytest
+functools32==3.2.3.post2;python_version=='2.7'
```

Comment:
Did you add these environment markers manually or pip-compile did?

Reply:
I did. I wasn't aware pip-compile could 👀

Reply:
I don't think it can